api/src/main/proto/bio/terra/tanagra/underlay/mapping.proto

```diff
@@ -0,0 +1,74 @@
+// Messages mapping logical entities to physical tables
```

Comment:
How should I be reading this: does this mapping work for BigQuery datasets only, or is this meant to support different backends? If it is specific to bigquery (and we might one day support others), should this be indicated in the package structure somewhere?

**Update**: Reading on - it seems this is meant to be generic across potentially different backends - will review with this in mind.

Reply:
Yep, added more to this comment.